src/components/common/NewsList.tsx

```diff
@@ -15,15 +16,15 @@ function NewsList(props: NewsListProps) {
 
   return (
     <StNewsContainer>
-      {newsList.map(({ id, title, channel, category, date }) => (
+      {newsList.map(({ id, title, category, channel, thumbnail, reportDate }) => (
         <StNewsWrapper key={id} onClick={() => router.push(`/learn/${id}`)}>
-          <StThumbnailContainer>
-            <StThumbnail />
+          <StThumbnail>
+            <ImageDiv className="thumbnail" src={thumbnail} layout="fill" alt="thumbnail" />
             <Like />
-          </StThumbnailContainer>
+          </StThumbnail>
           <StTitle>{title}</StTitle>
           <StInfo>
-            {channel} | {category} | {date}
+            {channel} | {category} | {reportDate}
```

Comment:
요거 서버에서 2022-07-16 이런 식으로 오기 때문에 우리가 바꿔줘야 해요!
```suggestion
            {channel} | {category} | {reportDate.replaceAll('-', '.')}
```

Reply:
허걱스!! 꼼꼼하게 봐줘서 고맙습니다!!